### DISCUSSION TITLE
why if login not login and need verify

### DISCUSSION THREAD

**Discussion by hdidjsh** [vercel/community#4081] [Account Issue] [answered]
### Summary

Why, when I try to register with my github account, it tells me to verify your number, but when I enter my number, the code is not sent and the following text appears in the image
![photo_2023-09-15_01-07-10](https://github.com/vercel/community/assets/144925988/89903d0a-2684-41f9-b5c6-b963372884e3)
if add number and prese submit :
![image](https://github.com/vercel/community/assets/144925988/41fc902c-f35d-462e-91c7-ed938767e1cb)

please help me

### Steps to Reproduce

Why, when I try to register with my github account, it tells me to verify your number, but when I enter my number, the code is not sent and the following text appears in the image
![photo_2023-09-15_01-07-10](https://github.com/vercel/community/assets/144925988/89903d0a-2684-41f9-b5c6-b963372884e3)
if add number and prese submit :
![image](https://github.com/vercel/community/assets/144925988/41fc902c-f35d-462e-91c7-ed938767e1cb)

please help me

**Comment by amyegan** [comment, marked as answer]
Hey @hdidjsh. I assume you were using a fake phone number for demonstration purposes. If you are using a real phone number and still getting that error message, please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will open a ticket with the team that can help with login and verification issues. We can't safely discuss account information here in public because of security concerns.
